s3tests_boto3/functional/test_s3.py

```diff
@@ -2817,6 +2817,53 @@ def test_post_object_upload_size_below_minimum():
 
     r = requests.post(url, files=payload, verify=get_config_ssl_verify())
     assert r.status_code == 400
+    assert r.error_code == 'EntityTooSmall'
+
+def test_post_object_upload_size_rgw_chunk_size_bug():
+    # TODO: if this value is different in Teuthology runs, this would need tuning
+    _rgw_max_chunk_size = 4 * 2**20 # 4MiB
+    min_size = _rgw_max_chunk_size * 2
+    max_size = _rgw_max_chunk_size * 3
```

Comment:
we try to keep rgw specifics out s3-tests, but i'm happy with the way you've done this 👍 it exercises a corner case for rgw, but should succeed regardless of the server. since it mentions an rgw bug, please add a comment that links to https://tracker.ceph.com/issues/58627

Reply:
Done